### PR TITLE
Add support for @propertyWrapper, @MainActor, @globalActor

### PR DESCRIPTION
**Pull request by AvdLee**
New types that didn't exist in Prism yet, but we need them! 🎉

### CODE REVIEW

File: components/prism-swift.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ Prism.languages.swift = Prism.languages.extend('clike', {
 	'keyword': /\b(?:actor|as|associativity|async|await|break|case|catch|class|continue|convenience|default|defer|deinit|didSet|do|dynamic(?:Type)?|else|enum|extension|fallthrough|final|for|func|get|guard|if|import|in|infix|init|inout|internal|is|lazy|left|let|mutating|new|none|nonisolated|nonmutating|operator|optional|override|postfix|precedence|prefix|private|protocol|public|repeat|required|rethrows|return|right|safe|self|Self|set|some|static|struct|subscript|super|switch|throws?|try|Type|typealias|unowned|unsafe|var|weak|where|while|willSet|__(?:COLUMN__|FILE__|FUNCTION__|LINE__))\b/,
 	'number': /\b(?:[\d_]+(?:\.[\de_]+)?|0x[a-f0-9_]+(?:\.[a-f0-9p_]+)?|0b[01_]+|0o[0-7_]+)\b/i,
 	'constant': /\b(?:nil|[A-Z_]{2,}|k[A-Z][A-Za-z_]+)\b/,
-	'atrule': /@\b(?:IB(?:Outlet|Designable|Action|Inspectable)|class_protocol|exported|noreturn|NS(?:Copying|Managed)|objc|UIApplicationMain|auto_closure)\b/,
+	'atrule': /@\b(?:IB(?:Outlet|Designable|Action|Inspectable)|class_protocol|exported|globalActor|noreturn|NS(?:Copying|Managed)|MainActor|objc|propertyWrapper|UIApplicationMain|auto_closure)\b/,
```
Comment (RunDevelopment) [inline components/prism-swift.js:22]: M < N

```suggestion
	'atrule': /@\b(?:IB(?:Outlet|Designable|Action|Inspectable)|class_protocol|exported|globalActor|MainActor|noreturn|NS(?:Copying|Managed)|objc|propertyWrapper|UIApplicationMain|auto_closure)\b/,
```

**Review by RunDevelopment — CHANGES_REQUESTED** [review]
One minor nit. Apart from that it's gtg. Thank you @AvdLee!

**Comment by RunDevelopment** [comment]
Also, please use backticks around `@propertyWrapper` (and others) in commit messages, comments, and titles. Otherwise, GitHub will ping these people. Let's try to not annoy them :)

**Comment by AvdLee** [comment]
> Also, please use backticks around @propertyWrapper (and others) in commit messages, comments, and titles. Otherwise, GitHub will ping these people. Let's try to not annoy them :)

Oh, oops! didn't think about that!

**Comment by RunDevelopment** [comment]
The build still fails. Could you please rebuild Prism?

**Comment by AvdLee** [comment]
> The build still fails. Could you please rebuild Prism?

I've never done that before, my previous PR succeeded as well w/o rebuilding Prism 🤔 
Is there any guidance for this? I'm not really experienced in deploying such packages

**Comment by RunDevelopment** [comment]
Just run `npm ci && npm run build` and commit the changes.

**Comment by AvdLee** [comment]
Thanks, that was actually easy. Pushed the changes, let's see if CI is happy now!

**Comment by RunDevelopment** [comment]
Thank you for contributing @AvdLee!

**Pull request merged**